contributor_docs/README.md

```diff
@@ -36,10 +36,17 @@ Head over to [this link](./contributor_guidelines.md) where you will be guided o
 
 Most of the time we will stick with this workflow quite strictly and, especially if you have contributed to other projects before, it may feel like there are too many hoops to jump through for what may be a simple contribution. However, the steps above are aimed to make it easy for you as a contributor and for stewards/maintainers to contribute meaningfully, while also making sure that you won't be spending time working on things that may not be accepted for various reasons. The steps above will help ensure that any proposals or fixes are adequately discussed and considered before any work begin, and often this will actually save you (and the steward/maintainer) time because the PR that would need additional fixing after review, or outright not accepted, would happen less often as a result.
 
-**We see contributing to p5.js as a learning opportunity** and we don't measure sucess by only looking at the volume of contributions we received. There is no time limit on how long it takes you to complete a contribution, so take your time and work at your own pace (we may check in after a long period of inactivity). Ask for help from any of the stewards or maintainers if you need them and we'll try our best to support you.
+**We see contributing to p5.js as a learning opportunity** and we don't measure sucess by only looking at the volume of contributions we received. There is no time limit on how long it takes you to complete a contribution, so take your time and work at your own pace (we may check in after a long period of inactivity). Ask for help from any of the stewards or maintainers if you need them and we'll try our best to support you. For information related to area stewards or general maintenance of p5.js GitHub repository, please check out the [steward guidelines](./steward_guidelines.md).
 
 ## Non-source code contribution
-There are many more ways to contribute to p5.js through non-source code contribution than can be exhaustively listed here, some of the ways may also involve working with some of the p5.js repositories (such as adding examples, writing tutorials for the website, etc.). Depending on what the planned contribution is, we may be able to support you in different ways so do reach out to us via any channel available to you (email, social media, [Discourse forum](https://discourse.processing.org/c/p5js/10), Discord, etc).
+There are many more ways to contribute to p5.js through non-source code contribution than can be exhaustively listed here. Some of the ways may also involve working with some of the p5.js repositories (such as adding examples, writing tutorials for the website, etc.). Depending on what the planned contribution is, we may be able to support you in different ways so do reach out to us via any channel available to you (email, social media, [Discourse forum](https://discourse.processing.org/c/p5js/10), Discord, etc). Here are just some ways you can contribute:
```

Comment:
Hi @sarahciston, we mentioned earlier that documentation is part of Source-code contribution, so we will need to move the documentation related text up. What do you think?

CC @limzykenneth

Reply:
I agree, and in that case we'd want to provide more introduction to documentation contributions there as well, right? On the website contribution page it says, "Document. Everyone loves helpful instructions. Check out the p5.js [GitHub](https://github.com/processing/p5.js) on how to get started with creating examples, improving reference pages, and writing tutorials." So where can we point to specifically for showing how to get started with writing reference pages and docs?

Reply:
We can point them to Contribute page on the new website

Reply:
On the Contribute page on the new website it points them to the GitHub contributor docs. Is there a place in the docs that addresses creating documentation specifically? Maybe we could point them to that? If not, perhaps that's something we could build in the future if it's desirable, but may need to remove the description of the pointer from the website if we don't have info on how to get started writing documentation yet. What do you both think? @Qianqianye @limzykenneth

Reply:
We have a doc about how to write referenece, and upcoming doc about how to write examples. We will have new links when the new website is updated. Maybe we can just mention the doc generically without links for now?